feature-libs/customer-ticketing/styles/components/_customer-ticketing-dialog.scss

```diff
@@ -1,4 +1,4 @@
-%cx-customer-ticketing-reopen-dialog {
+%cx-customer-ticketing-dialog {
```

Comment:
I understand re-using the stylesheet, but I don't think the button should be taking the full width based of the wireframes
![Screen Shot 2022-08-25 at 3 33 16 PM](https://user-images.githubusercontent.com/29099466/186752482-2befe120-bdf9-4e60-ade3-f5ba69cf83c5.png)